storage/command.h

```diff
@@ -0,0 +1,46 @@
+#ifndef STORAGE_COMMAND_H_
+#define STORAGE_COMMAND_H_
+
```

Comment:
The filename `storage/command.h` looks like an abstraction to invoke commands for specific storage, and we shall not expose the implementation information of Redis. Instead, can you move the Redis-only parts into C source rather than header files?

Reply:
Key-value storage is necessary for tangle-accelerator use case, and there is no visible reason why SQL or query statements are needed by design. Let's abstract the operations accordingly.